recurly/__init__.py

```diff
@@ -689,6 +689,11 @@ class Item(Resource):
         'deleted_at',
     )
 
+    def reactivate(self):
```

Comment:
Let's add a comment to the top of this method explaining that it can be removed when the action starts to get rendered on the server.